pyre/bench/synth/foriter_operator_index_replay_regression.py

```
@@ -0,0 +1,40 @@
+# call is the arm being admitted; the object call must stay opaque.  Admitting
+# the object call too would let the replay run `__index__` a second time, and
+# `hits` would read N + 1.
+
+from operator import index as _index
+
+N = 5000
+hits = [0]
+
+
+class C:
+    def __index__(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__index__`

Add return type annotation: `int`

(ANN204)